tools/pd-control.md

```diff
@@ -263,6 +264,8 @@ The configuration above is global. You can also tune the configuration by config
 
 - `disable-location-replacement` is used to disable the isolation level check. When you set it to `true`, PD does not improve the isolation level of Region replicas by scheduling.
 
+- `disable-namespace-relocation` is used to disable relocation region to stores of its namespace. When you set it to `true`, PD does not move regions to stores where they belong to.
```

Comment:
How about `the region relocation`?

Reply:
Ha? You mean `disable region relocation to the store of its namespace` ?

Reply:
yep